fasm/__init__.py

```diff
@@ -10,12 +10,18 @@
 # SPDX-License-Identifier: ISC
 
 from __future__ import print_function
-import textx
 import os.path
 import argparse
 from collections import namedtuple
 import enum
+from ctypes import *
+import os
+from sys import byteorder
 
+here = os.path.dirname(os.path.realpath(__file__))
```

Comment:
I think the ctypes stuff should be isolated in another python file where any memory handling is dealt with.  E.g. call into the C interface, copy to a python string, free the memory.  Then you can write a docstring around the wrapper function, etc.

Reply:
Done.